src/extension/tests/helpers/AssessmentSummary.json

```diff
@@ -0,0 +1,72 @@
+{
+    "assessmentActivityId": "d7e26b52-c18e-46ec-a44b-8cb7bd7a62a3",
```

Comment:
Please qualify the file name with the specific test it's intended to support. We don't want future test code lazy-leaning on test artifacts meant for specific tests resulting in cascading test failures if something needs to be tweaked for one test (or a closely related group of tests). Same comment for the other json.